src/data/apiMetadata.js

```diff
@@ -1,3187 +1,3187 @@
 export default [
   {
```

Comment:
Why these changes? Is there an issue with ordering?

Reply:
think it's just prettier changing the formatting?